### DISCUSSION TITLE
Combair SL (Temperature)

### DISCUSSION THREAD

**Discussion by ColossusCH** [siku2/hass-vzug#21] [Device Support]
Hello all,
first of all, thank you very much for this extremely helpful integration.
I still have one question:
I have a Combair SL.
I have the following entities but no display of the current temperature.
Can this still be integrated or am I just too incompetent to find this?

sensor.combair_steam_sl_letzte_benachrichtigung
sensor.combair_steam_sl_program_ende_rohwert
sensor.combair_steam_sl_programm
sensor.combair_steam_sl_programm_ende
sensor.combair_steam_sl_zustand
button.combair_steam_sl_nach_updates_suchen
update.combair_steam_sl_update

Many thanks

**Comment by siku2** [comment]
Can you confirm whether you can see the temperature on the website or the app?
